script.js

```diff
@@ -0,0 +1,69 @@
+const addButton = document.getElementById('addButton');
+const booksList = document.getElementById('booksList');
+const addAuthor = document.getElementById('addAuthor');
+const addTitle = document.getElementById('addTitle');
+
+let books;
+
+if (localStorage.getItem('books') === null) {
+  books = [];
+} else {
+  books = JSON.parse(localStorage.getItem('books'));
+}
+
+function displayBooks() {
+  books.forEach((book) => {
+    const li = document.createElement('li');
+    li.innerHTML = `
+    <h2>${book.title}</h2>  
+    <p>${book.author}</p>  
+    <button class="remove">remove</button>
+    <hr>`;
+    booksList.append(li);
+  });
+}
+
+function addBook(e) {
+  e.preventDefault();
+  const title = addTitle.value;
+  const author = addAuthor.value;
+  const book = {
+    title,
+    author,
+  };
+  book.id = Math.floor(Math.random() * 100);
+  books.push(book);
+  localStorage.setItem('books', JSON.stringify(books));
+  addTitle.value = '';
+  addAuthor.value = '';
+  const li = document.createElement('li');
+  li.classList.add('book');
+  li.dataset.id = book.id;
+  li.innerHTML = `
+    <h2>${book.title}</h2>  
+    <p>${book.author}</p>  
+    <button class="remove">remove</button>
+    <hr>`;
+  booksList.append(li);
+}
+
+function removeBook(elem) {
```

Comment:
- [ ] When 2 or more books with **same** _book name_ and _book author_  are stored and then a single entry having same details is removed all other entry with the same details also gets removed. You can prevent this by ensuring that each entry has a unique `id` or check if a  duplicate exists and prevent entry

Reply:
Thanks! Added a function to prevent duplicate entries.